packages/components/src/Subscription/Component.purs

```diff
@@ -167,19 +167,33 @@ render self@{ props: props@{ subscription: sub@{ package } } } =
         where
           asyncWrapper = AsyncWrapper.asyncWrapper
             { wrapperState: self.state.wrapperProgress
-            , readyView: actionsContainer $ [ pauseIcon
-                                            , if maybe true Array.null self.state.pausedSubscriptions
-                                              then mempty
-                                              else removeSubscriptionPauses
-                                            , temporaryAddressChangeIcon
-                                            , deliveryReclamationIcon
-                                            ]
+            , readyView: actionsContainer $ defaultActions
             , editingView: identity
-            , successView: \msg -> successContainer [ DOM.div { className: "subscription--update-success check-icon" }, foldMap successMessage msg  ]
-            , errorView: \err -> errorContainer [ errorMessage err, tryAgain ]
+            , successView: \msg -> actionsContainer $ append defaultActions $ [successWrapper msg]
```

Comment:
Just occurred to me, you can use `<>` instead of `append`, it is more common in our codebase and in general 😃